tests/client_auth_revocation.rs

```diff
@@ -179,7 +179,7 @@ fn ee_not_revoked_wrong_ku_ee_depth() {
     .unwrap()];
     assert_eq!(
         check_cert(ee, intermediates, ca, RevocationCheckDepth::EndEntity, crls),
-        Ok(())
+        Err(webpki::Error::UnknownIssuer)
```

Comment:
This means that these tests failed in the previous commit?

Reply:
Yup :ballot_box_with_check: